lib/puppet/type/user.rb

```diff
@@ -200,6 +200,12 @@ def sync
           well as the password.
         * Windows passwords can only be managed in cleartext, as there is no Windows API
           for setting the password hash.
+        * Windows password cannot be set to nil, and should not be set to an empty string.
+          While this may technically be allowed, with a non-default local system level
+          password complexity configuration, there is no way to verify that an existing
+          password is blank. Therefore each Puppet run will emit a change notice for any
+          Windows user configured with a blank password, even if the users existing
+          password was already blank.
 
```

Comment:
What does "set to nil" mean in the context of a Puppet manifest? `password => undef,`?